sdk/search/search-documents/test/node/searchClient.spec.ts

```diff
@@ -18,20 +18,21 @@ import { Hotel } from "../utils/interfaces";
 import { createIndex, populateIndex } from "../utils/setupIndex";
 import { delay } from "@azure/core-http";
 
-const TEST_INDEX_NAME = "hotel-live-test";
+const TEST_INDEX_NAME = "hotel-live-test1";
```

Comment:
should we generate this name randomly each time to avoid needing to coordinate between test files?

Reply:
Between 3 cases, I do not see this is required. Also, random generation might create problems with the recordings.

Reply:
Alright, let's keep it as is